includes/assets/web-stories-block/scss/main.scss

```diff
@@ -0,0 +1,22 @@
+:root {
+  --ws-font-size-title: 18px;
+  --ws-font-size-circle-title: 14px;
+  --ws-font-size-excerpt: 16px;
+  --ws-font-size-author-date: var(--ws-font-size-circle-title);
```

Comment:
?

Reply:
Custom properties so that some of the styling aspects can be modified easily, mainly the font sizes and meta colour.

Reply:
If you just set one variable to another, what is the point of this?

Reply:
There are use cases of using the one variable as another's value but maybe not highlighted or required here, I'll update it here. 😅